backend/djangoindia/db/models/event.py

```diff
@@ -26,32 +26,41 @@ class Event(BaseModel):
     venue = models.TextField(default="TBA",null=True, blank=True)
     city = models.CharField(max_length=255, default="TBA", null=True, blank=True)
     venue_map_link = models.TextField(null=True, blank=True)
-    event_start_date = models.DateTimeField(null=False, default=default_start_date)
-    event_end_date = models.DateTimeField()
+    start_date = models.DateTimeField(null=False, default=default_start_date)
+    end_date = models.DateTimeField()
     registration_end_date = models.DateTimeField(default=default_registration_end_date)
     event_mode = models.CharField(max_length=20,choices=EVENT_MODE_CHOICES,default=IN_PERSON)
 
     def clean(self):
-        if self.event_end_date and self.event_start_date:
-            if self.event_end_date <= self.event_start_date:
+        if self.end_date and self.start_date:
+            if self.end_date <= self.start_date:
                 raise ValidationError("Event end date must be after event start date.")
 
         super().clean()
 
     def __str__(self) -> str:
-        return f"{self.name} @ {self.city} ({self.event_start_date.date()})"
+        return f"{self.name} @ {self.city} ({self.start_date.date()})"

```

Comment:
Please elaborate on this @DevilsAutumn , back on django after a bit. I am sure there is reason we need that ending date func ?

Reply:
actually, start_date is a datetime field , so to extract just the date we use date func.